src/include/sof/lib/ams.h

```diff
@@ -24,6 +24,9 @@
 /* Wildcard for module_id and instance_id values */
 #define AMS_ANY_ID 0xFFFF
 
+/* AMS messages */
+extern const uint8_t LP_KEY_PHRASE_DETECTED_UUID[UUID_SIZE];
```

Comment:
why should an array be called in all capital letters?

Reply:
Please, see comment below